Sources/SmileID/Classes/Metadata/UIDeviceOrientationExtension.swift

```diff
@@ -0,0 +1,20 @@
+import UIKit
+
+extension UIDeviceOrientation {
+    var category: String {
+        switch self {
+        case .portrait, .portraitUpsideDown:
+            return "Portrait"
+        case .landscapeLeft, .landscapeRight:
+            return "Landscape"
+        case .faceUp:
+            return "FaceUp"
+        case .faceDown:
+            return "FaceDown"
+        case .unknown:
```

Comment:
And maybe also here lower-case `unknown`. Just to keep consistency in case we can't extract it.